### ISSUE TITLE
training loss chart

### ISSUE THREAD

**Issue by yeqiuyi** [BubblyYi/Coronary-Artery-Tracking-via-3D-CNN-Classification#5]
Can I have a look at your training loss chart? my direction loss can be reduced normally, but neither the radius nor proximity regression loss can be reduced. 
thank you very much！

**Comment by simi-chen** [comment]
Have you solved your problem? I have the same problem

**Comment by yeqiuyi** [comment]
@simi-chen no,I find it very strange, there is no problem with the code data for training and validation.

**Comment by lalalala123456789** [comment]
Can I know which Pytorch version do you use?

**Comment by BubblyYi** [comment]
Hi, Thank you for your interest in my project. For your question, I have the following suggestions. First of all, please consider the data distribution and the proportion of positive and negative samples. The key is whether the distribution of the training set and test set is consistent.

**Comment by zyx3216825** [comment]
可以给我一份你的数据吗，cta08官网不能下载啊

**Comment by renshengruxge** [comment]
> 可以给我一份你的数据吗，cta08官网不能下载啊

你好，请问数据集可以

> 可以给我一份你的数据吗，cta08官网不能下载啊

你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱2823212088@qq.com，谢谢！

**Comment by XuyangPan** [comment]
xuyangpan@yeah.net
你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱[xuyangpan@yeah.net](mailto:xuyangpan@yeah.net)，谢谢！

**Comment by XuyangPan** [comment]
你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱[xuyangpan@yeah.net](mailto:xuyangpan@yeah.net)，谢谢！

**Comment by XuyangPan** [comment]
> > 可以给我一份你的数据吗，cta08官网不能下载啊
> 
> 你好，请问数据集可以
> 
> > 可以给我一份你的数据吗，cta08官网不能下载啊
> 
> 你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱[2823212088@qq.com](mailto:2823212088@qq.com)，谢谢！

你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱[xuyangpan@yeah.net](mailto:xuyangpan@yeah.net)，谢谢！

**Comment by wacqkl** [comment]
你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱1551318215@qq.com，谢谢！

**Comment by phe121** [comment]
你好，请问你是否方便提供数据集，官网最近登录不进去没有办法下载数据集，如果方便的话请发送到我的邮箱2698056011@qq.com，谢谢！

**Comment by renshengruxge** [comment]
不好意思，我也没有找到数据集



---Original---
From: ***@***.***&gt;
Date: Fri, Jul 21, 2023 10:10 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [BubblyYi/Coronary-Artery-Tracking-via-3D-CNN-Classification]training loss chart (#5)




 
***@***.***，谢谢！
